src/components/Form/inputs/AddDebitCardForm.ts

```diff
@@ -0,0 +1,30 @@
+import type {Form} from '@src/types/onyx';
```

Comment:
Same case, we could move Form type to the same place where the form types will be e.g. `src/types/form`. It's strange to me to have specific Form types under a component folder, while the generic Form type is under a Onyx types folder.